ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -680,6 +693,144 @@ public void shouldNotIncludeSslPropertiesInListPropertiesOutput() {
         not(hasItems(KsqlConfig.SSL_CONFIG_NAMES.toArray(new String[0]))));
   }
 
+  @Test
+  public void shouldFailForInvalidTerminateClusterParameters() {
+    final Map<String, Object> properties = new HashMap<>();
+    properties.put(TerminateCluster.DELETE_TOPIC_LIST_PARAM_NAME, Collections.singletonList("FOO"));
+    final KsqlRequest ksqlRequest = new KsqlRequest(TerminateCluster.TERMINATE_CLUSTER_STATEMENT_TEXT, properties);
+    final KsqlResource testResource = TestKsqlResourceUtil.get(ksqlConfig, ksqlEngine);
+    final Response response = testResource.terminateCluster(ksqlRequest);
+    assertThat(response.getStatus(), equalTo(200));
+  }
+  @Test
+  public void shouldFailForMisSpelledTerminateClusterParameters() {
+    final Map<String, Object> properties = new HashMap<>();
```

Comment:
Can you look to use the same much more readable pattern as the tests above please.